kong/runloop/handler.lua

```diff
@@ -322,8 +322,14 @@ local function register_events()
     local operation = data.operation
     local target = data.entity
 
-    -- => to balancer update
-    balancer.on_target_event(operation, target)
+    local upstream_id = target.upstream.id
```

Comment:
Is that possible that `target.upstream` or `target.upstream.id` are somehow `nil`?

Reply:
I don't think so. A target always has an upstream right?

Reply:
Yes, just checked, it's required